unit-tests/run-unit-tests.py

```diff
@@ -179,11 +207,15 @@ def cat( filename ):
     manifestfile = dir + '/CMakeFiles/TargetDirectories.txt'
 else:
     manifestfile = dir + '/../CMakeFiles/TargetDirectories.txt'
-for manifest_ctx in grep( r'(?<=unit-tests/build/)\S+(?=/CMakeFiles/test-\S+.dir$)', manifestfile ):
 
+for manifest_ctx in grep( r'(?<=unit-tests/build/)\S+(?=/CMakeFiles/test-\S+.dir$)', manifestfile ):
+    
     testdir = manifest_ctx['match'].group(0)                    # "log/internal/test-all"
     testparent = os.path.dirname(testdir)                       # "log/internal"
-    testname = 'test-' + testparent.replace( '/', '-' ) + '-' + os.path.basename(testdir)[5:]   # "test-log-internal-all"
+    if testparent != "":
```

Comment:
I believe an empty string will return False so you should be able to do:
```
if testparent:
```
Can you try?

Reply:
done